app/editor/src/features/content/form/ContentStoryForm.tsx

```diff
@@ -61,34 +61,25 @@ export const ContentStoryForm: React.FC<IContentStoryFormProps> = ({
         size: fileReference.size,
       } as IFile)
     : undefined;
-  const [race, setRace] = React.useState(false); // Required to handle race condition when initializing the stream.
```

Comment:
Is this no longer required?

Reply:
I don't think we need it anymore and it is part of the cause for the current issue, but if this is for something else which I'm not aware of, please let me know. Thanks